content/en/docs/instrumentation/java/automatic/annotations.md

```diff
@@ -12,17 +12,17 @@ custom code without doing too much code change.
 
 ## Dependencies
 
-You'll need to add a dependency on the `opentelemetry-extension-annotations`
+You'll need to add a dependency on the `opentelemetry-instrumentation-annotations`
 library to use the `@WithSpan` annotation.
 
 ### Maven
 
 ```xml
 <dependencies>
   <dependency>
-    <groupId>io.opentelemetry</groupId>
-    <artifactId>opentelemetry-extension-annotations</artifactId>
-    <version>1.16.0</version>
+    <groupId>io.opentelemetry.instrumentation</groupId>
+    <artifactId>opentelemetry-instrumentation-annotations</artifactId>
+    <version>1.17.0</version>
```

Comment:
it's `1.17.0-alpha`, see https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/6494

Reply:
Oops! @jack-berg - can you submit an update?

Reply:
Oops! Fixed in #1651.